lms/templates/courseware/progress.html

```diff
@@ -104,6 +105,49 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
         <div class="grade-detail-graph" id="grade-detail-graph" aria-hidden="true"></div>
       %endif
 
+      %if credit_course is not None:
+        <div class="credit-eligibility">
+            <div class="credit-eligibility-container">
+                <div class="eligibility-heading">
+                    <h2>${_("Eligibility Requirements for credit")}</h2>
+                </div>
+                %if credit_course['eligibility_status'] == 'not_eligible':
+                    <span class="eligibility_msg">${student.username}, ${_("You are no longer eligible for this course")}</span>
+                %elif credit_course['eligibility_status'] == 'eligible':
+                    <span class="eligibility_msg">${student.username}, ${_("You have met the requirements for the credit. Please")}
+                        <a href="#">${_("Check your dashboard")}</a> ${_("for purchase.")}
+                    </span>
+                %elif credit_course['eligibility_status'] == 'partial_eligible':
+                    <span>${_("You have not yet met the requirements for credit")}</span>
```

Comment:
*student.username*, you have not yet met all of the requirements for course credit. (add student name; note capitalization and punctuation)